google-cloud-bigquerystorage/src/main/java/com/google/cloud/bigquery/storage/v1/ConnectionWorker.java

```diff
@@ -1372,16 +1375,21 @@ static final class AppendRequestAndResponse {
     // If a response is no longer expected this is set back to null.
     Instant requestSendTimeStamp;
 
+    // Mark where the traffic coming from. Used for internal logging purpose.
+    String fullTraceId;
```

Comment:
Why not retrieve the traceId from streamWriter.getFullTraceId()?

Reply:
ConnectionWorker does not have reference to individual stream writer

Reply:
Since multiple SW can share the same connecctionWorker

Reply:
I meant the streamWriter found in AppendRequestAndResponse. This is available for each request, right?

Reply:
You are right, done